mlir/test/Dialect/ArmSME/vector-legalization.mlir

```diff
@@ -646,3 +646,11 @@ func.func @negative_transpose_store_scalable_via_za__bad_source_shape(%vec: vect
   vector.transfer_write %tr, %dest[%i, %j] {in_bounds = [true, true]} : vector<[7]x2xf32>,  memref<?x?xf32>
   return
 }
+
+// -----
+
+// From: https://github.com/llvm/llvm-project/issues/118449 (check we don't crash).
+func.func @vector_mask_empty(%m0: vector<16x2xi1>, %arg1: vector<16x16xf32>) -> vector<16x16xf32> {
```

Comment:
1. I don't follow- the mask is not empty? (looking at test func name)
2. The convention in this file seems `%mask` rather than `%m0` for masks. Please update.
3. Replace `%arg1` with something more descriptive, consistent with what's already used in the file.